R/JoinKeys.R

```diff
@@ -473,3 +538,19 @@ join_key <- function(dataset_1, dataset_2, keys) {
     class = "JoinKeySet"
   )
 }
+
+#' @rdname join_key
+#' @export
+#'
+#' @details
+#' `primary_key()` will create a primary key for a dataset. It is equivalent to
+#' `join_key(...)` and omitting `dataset_2` argument or giving it the same name
+#' `dataset_1`.
+#'
+primary_key <- function(dataset_1, keys) {
```

Comment:
@averissimo what do you like better, extra public function `primary_key` or `join_key(dataset_1, dataset_2 = dataset_1)`?

It seems unnecessary

Reply:
I used `primary_key(...)` due to the example on the issue. I would prefer keeping `join_key(...)` to keep it simple.

I'm removing it :-)